languages.toml

```diff
@@ -708,9 +708,11 @@ grammar = "javascript"
 name = "typescript"
 scope = "source.ts"
 injection-regex = "(ts|typescript)"
-file-types = ["ts", "mts", "cts"]
 language-id = "typescript"
-shebangs = ["deno", "ts-node"]
+file-types = ["ts", "mts", "cts"]#, { glob = ".deno_repl_history" }, { glob = ".bun_repl_history" }]
+# Deno doesn't save hist-file by default: https://deno.land/x/manual@v1.36.4/tools/repl.md?source=#deno_repl_history
+# Bun doesn't (yet) have a standard REPL history file (this info wasn't verified)
```

Comment:
This change seems unnecessary - this can be a follow-up PR when/if deno and bun add history files